Fitness/Views/Home/GymListItemCell.swift

```diff
@@ -0,0 +1,182 @@
+        contentView.layer.borderColor = UIColor.fitnessLightGrey.cgColor
+        contentView.layer.borderWidth = 0.5
+
+        // SHADOWING
+        contentView.layer.shadowColor = UIColor.fitnessBlack.cgColor
+        contentView.layer.shadowOffset = CGSize(width: 0.0, height: 15.0)
+        contentView.layer.shadowRadius = 5.0
+        contentView.layer.shadowOpacity = 0.1
+        contentView.layer.masksToBounds = false
+        let shadowFrame = contentView.frame.inset(by: UIEdgeInsets(top: 0, left: 0, bottom: 9, right: -3))
+        contentView.layer.shadowPath = UIBezierPath(roundedRect: shadowFrame, cornerRadius: 5).cgPath
+
+        setupViews()
+        setupConstraints()
+    }
+
+    // MARK: - Public configure
+    override func configure(for gym: Gym) {
+        super.configure(for: gym)
+
+        // Set gym name
+        locationNameLabel.text = gym.name
+
+        // Set gym status
+        let changingSoon = gym.isStatusChangingSoon()
+        if gym.isOpen {
+            if changingSoon {
+                statusLabel.textColor = .fitnessOrange
+                statusLabel.text = "Closing soon"
+            } else {
+                statusLabel.textColor = .fitnessGreen
+                statusLabel.text = "Currently open"
+            }
+        } else {
+            if changingSoon {
+                statusLabel.textColor = .fitnessOrange
+                statusLabel.text = "Opening soon"
+            } else {
+                statusLabel.textColor = .fitnessRed
+                statusLabel.text = "Closed"
+            }
+        }
+
+        // Set gym hours
+        hoursLabel.text = getHoursString(from: gym)
+    }
+
+    // MARK: - Private helpers
+    private func getHoursString(from gym: Gym) -> String {
+        let now = Date()
+        let isOpen = gym.isOpen
+
+        let gymHoursToday = gym.gymHoursToday
+        let gymHoursTomorrow = gym.gymHours[now.getIntegerDayOfWeekTomorrow()]
+
+        let format: String
+
+        if now > gymHoursToday.closeTime {
+            if Calendar.current.component(.minute, from: gymHoursTomorrow.openTime) == 0 {
+                format = "h a"
+            } else {
+                format = "h:mm a"
+            }
+            if gym.closedTomorrow {
+                return "Closed Tomorrow"
+            } else {
+                return "Opens at \(gymHoursTomorrow.openTime.getStringOfDatetime(format: format))"
+            }
+        } else if !isOpen {
+            if Calendar.current.component(.minute, from: gymHoursToday.openTime) == 0 {
+                format = "h a"
+            } else {
+                format = "h:mm a"
+            }
+            return "Opens at \(gymHoursToday.openTime.getStringOfDatetime(format: format))"
+        } else {
+            if Calendar.current.component(.minute, from: gymHoursToday.closeTime) == 0 {
+                format = "h a"
+            } else {
+                format = "h:mm a"
+            }
```

Comment:
that same helper function could be used here as well, I think it should definitely be factored out